main.go

```diff
@@ -71,3 +114,31 @@ func handleSigterm(stopChan chan struct{}) {
 	log.Infoln("received SIGTERM. Terminating...")
 	close(stopChan)
 }
+
+func newClient() (*kubernetes.Clientset, error) {
+	var (
+		config *rest.Config
+		err    error
+	)
+
+	kubeconfig := clientcmd.RecommendedHomeFile
```

Comment:
this should be set if `--in-cluster=false` or something. Otherwise buildconfigfromflags will always resort to local .kube/config file